src/settings/flash.rs

```diff
@@ -30,7 +30,7 @@ impl sequential_storage::map::Key for SettingsKey {
     }
 }
 
-pub struct SerialSettingsPlatform<C, const Y: usize> {
+pub struct SerialSettingsPlatform<C> {
```

Comment:
A lot of this is copy-paste between booster/stabilizer/thermostat-eem etc [waiting for refactor](https://github.com/quartiq/stabilizer/issues/909).
Could you check that 
https://github.com/quartiq/stabilizer/blob/main/src/settings.rs matches where it overlaps whith this code here?